docs: drop weasel words (#22165)
renovatebot/renovate: docs/usage/getting-started/private-packages.md

Currently the preferred way to configure `github.com` credentials for self-hosted Renovate is:

> - Create a read-only Personal Access Token (PAT) for a `github.com` account. This can be any GitHub account, it might be better to create an "empty" account for this purpose.

Author: ```suggestion
- Create a read-only Personal Access Token (PAT) for a `github.com` account. This can be any GitHub account, but we recommend you create an "empty" account for this purpose.
```

How about this?
